RELEASE.rst

```diff
@@ -71,26 +71,21 @@ permissions.
 * Commit these changes using ``git commit``, with a commit message like
   ``Preparing release v0.0.1``.
 
-* Run ``git clean -fxd`` to be sure you are building in a clean environment.
-
-* Run ``./bootstrap.sh``: you need `automake`, `autoconf` and `libtool`
-  installed.  If no errors appear, this will create a new `./configure`
-  file.
-
-* Run ``./configure``, which should create a `Makefile` in the top level
-  directory and in ./src, as well as a `config.h` file in the top level.
+* Run ``meson setup builddir``: you need `meson` installed.  If no errors appear,
+  this will create a `build.ninja` file in the build directory, as well as a
+  `config.h`.
   To avoid possible trouble, check that in the latter, `PACKAGE_VERSION`
   is set correctly.
 
-* Run ``make check``, which will build the library and run the tests -
+* Run ``ninja -C builddir test``, which will build the library and run the tests -
```

Comment:
How about replacing this with `meson test -C builddir`? It calls the same thing, but can be used directly with `git bisect` according to the man page.

Reply:
The two commands are basically the same thing.
- `ninja test` first builds all artifacts, then executes `meson test --no-rebuild` to avoid recursion. It's designed to be a simple alias target that does what you probably want to do
- `meson test` will first try to run `ninja` to rebuild all targets, but can't rebuild when using the Visual Studio or xcode backends
  - `meson test` is a power user's frontend, and the main advantage of using it directly is that it has options to control test timeouts, wrap the tests with valgrind or gdb, select a subset of tests to run, etc.

Either one is fine, so we can use whichever one you prefer.

Actually for consistency with meson dist we might as well refer to meson test...